tests/test_config.py

```diff
@@ -75,6 +76,7 @@ def test_env_vars(self):
         expected_value["feature_flags"]["variables"] = True
         self.assertEqual(expected_value, value)
 
+    @unittest.skipIf(sys.platform.startswith("win"), "Test skipped on Windows")
     def test_file(self):
```

Comment:
This doesn't work on Windows?

If we want users to be able to use Windows, the ability to use configuration is important.

Reply:
it does not work on windows:

https://github.com/formiclabs/deeporigin-client/actions/runs/8989133804/job/24691537723

Reply:
Because is small, I think we should try to fix it in another ticket

The error is below. It seems like we're not releasing a handle to a file, which likely isn't really a Windows issue.
```
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmp6nmcyxfs'
```

The error comes from this line. This line simply cleans up a temporary file.
https://github.com/formiclabs/deeporigin-client/blob/7a4897a2a9b7c18abdc06ce441afa90931aa0535/tests/test_config.py#L120

The other errors also stem from the same file access issue. That said, I don't think we should try to make installing variables/secrets work for Windows. So those tests should be skipped.